bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/model/DefaultSemanticTagProvider.java

```diff
@@ -39,59 +39,56 @@ public DefaultSemanticTagProvider() {
         defaultTags.add(DefaultSemanticTags.LOCATION);
```

Comment:
The topmost tags are sorted Equipment => Location => Point =>, Property.

But by contrast your sort algorithm seems to be Location => Property => Point => Equipment.

Personally I suggest that both should be sorted Location => Point => Property => Equipment.

Reply:
Yeah this was the original ordering. I'll adjust